test/evaluation/evaluationClientWrapper.spec.ts

```diff
@@ -169,25 +169,25 @@ describe('evaluate', () => {
 
   it('should return ok if uri in vp matches at least one of input_descriptor\'s uris', function() {
     const pdSchema: PresentationDefinition = getFile('./test/dif_pe_examples/pd/pd-simple-schema-age-predicate.json').presentation_definition;
-    const vpSimple = getFile('./test/dif_pe_examples/vp/vp-simple-age-predicate.json');
+    const vpSimple: VerifiablePresentation = getFile('./test/dif_pe_examples/vp/vp-simple-age-predicate.json');
     pdSchema.input_descriptors[0].schema.push({ uri: 'https://www.w3.org/TR/vc-data-model/#types1' });
     const evaluationClientWrapper: EvaluationClientWrapper = new EvaluationClientWrapper();
     const evaluationClient: EvaluationClient = evaluationClientWrapper.getEvaluationClient();
     vpSimple.holder = HOLDER_DID;
-    const evaluationResults = evaluationClientWrapper.evaluate(pdSchema, new VP(vpSimple));
+    const evaluationResults = evaluationClientWrapper.evaluate(pdSchema, vpSimple);
     const errorResults = evaluationClient.results.filter(result => result.status === Status.ERROR);
     expect(errorResults.length).toEqual(0);
     expect(evaluationResults).toEqual(success);
   });
 
   it('should return error if uri in verifiableCredential doesn\'t match', function() {
     const pdSchema: PresentationDefinition = getFile('./test/dif_pe_examples/pd/pd-simple-schema-age-predicate.json').presentation_definition;
-    const vpSimple = getFile('./test/dif_pe_examples/vp/vp-simple-age-predicate.json');
-    vpSimple.verifiableCredential[0]['@context'] = 'https://www.w3.org/TR/vc-data-model/#types1';
+    const vpSimple: VerifiablePresentation = getFile('./test/dif_pe_examples/vp/vp-simple-age-predicate.json');
```

Comment:
we had to move these to TypeScript files from json files.

ps. all json example should be moved to ts.